.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
```

Comment:
What is this?

Reply:
I think the prior non-native version of dependabot used version 1 of this config format.
The current native dependabot docs provide examples the now specify using version 2.

https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#about-the-dependabotyml-file

Reply:
In case you were just asking about what is dependabot:
https://dependabot.com/

Reply:
I don't really understand what this is doing for us. Is this updating the underlay repos file or something?

Reply:
It does stuff like this for us, updating the Dockerfile, thus updating the CI image:
https://github.com/ruffsl/docker_images/pull/29

Reply:
I guess I don't really understand, why should we go that far out of our way to use specifically dated docker images for testing / ubuntu bases? That seems like a huge headache